DiztinGUIsh/window/dialog/ImportROMDialog.cs

```diff
@@ -2,7 +2,6 @@
 using System.ComponentModel;
```

Comment:
note for future people: this form contained a lot of controller / business logic, in this PR it's been refactored out so it can more easily be unit tested.